group_vars/mep/vars.yml

```diff
@@ -17,8 +17,8 @@ paris_overlay: 'https://tiles.arcgis.com/tiles/4Ko8f1mCWFLyY4NV/arcgis/rest/serv
 # These include the path for rh-python35 and nodejs8
 # Paths are picky and spaces for any YAML multiline syntax cause issues
 # These are defaults for most projects and should be overriden as necessary.
-path: '/opt/rh/rh-python35/root/usr/bin:/opt/rh/rh-nodejs8/root/usr/bin:{{ ansible_env.PATH }}'
-ld_library_path: '/opt/rh/rh-python35/root/usr/lib64:/opt/rh/rh-nodejs8/root/usr/lib64{% if ansible_env.LD_LIBRARY_PATH is defined %}:{{ ansible_env.LD_LIBRARY_PATH }}{% endif %}'
+path: '/opt/rh/rh-python35/root/usr/bin:/opt/rh/rh-nodejs8/root/usr/bin:/opt/rh/rh-git227/root/bin/:{{ ansible_env.PATH }}'
+ld_library_path: '/opt/rh/rh-python35/root/usr/lib64:/opt/rh/rh-nodejs8/root/usr/lib64:/opt/rh/httpd24/root/usr/lib64{% if ansible_env.LD_LIBRARY_PATH is defined %}:{{ ansible_env.LD_LIBRARY_PATH }}{% endif %}'
```

Comment:
should we also do MEP prod?

Reply:
I don't see any path overrides for mep prod or qa; they're both using this config (since qa actually matches prod for mep, unlike ppa)